lib/ui/questions/survey_question_view.dart

```diff
@@ -77,17 +78,34 @@ class _SurveyQuestionViewState extends ConsumerState<SurveyQuestionView> {
     );
   }
 
-  Widget _buildAnswers(SurveyQuestionUiModel question) {
-    switch (question.displayType) {
-      case QuestionDisplayType.dropdown:
-        return AnswerDropdown(
-          key: SurveyQuestionsWidgetId.answersDropdown,
-          answers: question.answers,
-        );
-      default:
-        return const SizedBox.shrink();
-    }
-  }
+  Widget _buildAnswers(SurveyQuestionUiModel question) =>
```

Comment:
since the answer types are growing up, can we create a separate widget for Answers? 🙏

Reply:
@manh-t Makes sense 👍 Can I refactor on the next PR? because we are still having other question types to implement 😄

Reply:
@hoangnguyen92dn okay 🙆‍♂️